modules/core/src/main/java/org/apache/ignite/internal/commandline/ActivateCommand.java

```diff
@@ -49,7 +50,7 @@
         try (GridClient client = Command.startClient(cfg)) {
             GridClientClusterState state = client.state();
 
-            state.active(true);
+            state.state(ACTIVE, true);
```

Comment:
Why `forceDeactivation` should be `true` for cluster activate?

Reply:
Same. Previous invocations were considered forced. I think we should not change it where not required.